app/controllers/images/votes/anonymity_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
For future reference:
We should get rid of anonymous voting (both for images and proposed names).
See https://www.pivotaltracker.com/story/show/153902213, https://www.pivotaltracker.com/story/show/155106697
For image votes @pellaea wanted to cut the cord, I wanted to phase it out. 
If cutting the cord, this might be a good opportunity?
But likely beyond the scope of this PR.

Reply:
I like it. I'm trying to keep removing/altering functionality outside these PRs, but i hope they make alterations easier to achieve. 

To me that's a nice thing about the controller (and eventual service object) split up — it's hopefully more evident, if we're taking out functionality, what the likely boundaries of its effects are.